### PR TITLE
Milestone four: Logic

### PR DESCRIPTION
**Pull request by phylischumba**
### Add logic App

- Added logic to the calculator application
- Fixed linter errors

### CODE REVIEW

File: src/logic/operate.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+/* eslint-disable no-unused-vars */
+import Big from 'big.js';
+
+export default function operate(numberOne, numberTwo, operation) {
+  let result = 0;
```
Comment (mosaaleb) [inline src/logic/operate.js]: - [x] Should this function return the result? It sets the result but returns an undefined value. 
We could also get rid of the result variable, and return the result directly within the if-else body.

**Review by mosaaleb — CHANGES_REQUESTED** [review]
#### Well done @Phylis05 :clap:

You have done all the requirements. Let's make sure before moving on that we return the result from the `operate` method.

Please handle this requirement and submit a new code review.

##### Muhammad Ebeid

**Review by jofer86 — APPROVED** [review]
## `const PROJECT_STATUS = "APPROVED"` 🚀 🚀 🚀 
## :wave: :wave: Hello! I'm Jorge, and  I will be conducting this code review. Let me get a closer look 🔎 🔍 🔎 🔍 🕵🏼 
- Changes marked as **[OPTIONAL]** are not hard requirements but are nice to have.
- All to the required changes were completed as requested, this milestone is done, you  may move along.
--------------------------------------------------------------------
**Rock on**  🤘🏼 
**_Jorge Fernando Rincón Figueroa._** 
Technical Support Engineer.

**Comment by phylischumba** [comment]
Hey @jofer86, thanks for the code review and approving this PR.

**Pull request merged**